config/jobs/kubernetes/sig-k8s-infra/trusted/sig-testing-branchprotector.yaml

```diff
@@ -24,6 +24,7 @@ periodics:
       - --confirm
       - --github-endpoint=http://ghproxy.default.svc.cluster.local
       - --github-endpoint=https://api.github.com
+      - --github-hourly-tokens=1000 # Up the rate limit from default (300) to 1000
```

Comment:
This is going to potentially cause other prow services to run out of quota, right? (let's be prepared to revert)

@upodroid @cjwagner what about moving to github app now? does it have to be wholesale with all of prow or can we migrate just this job / the k8s-infra-trusted-cluster gh api cache?

Reply:
This token is for @k8s-github-robot which should only be doing this/peribolos/label sync as far as I'm aware. Peribolos caps at 3k tokens, branch protector at 1000 (this change), and label sync at 300. Hopefully this won't cause issues but you never know.

Reply:
Being that the GH app cap is 15k/hour, we might be able to just move these three things over to it, cache or not?

Reply:
> This token is for @k8s-github-robot which should only be doing this/peribolos/label sync as far as I'm aware.

excellent
/hold cancel